### PR TITLE
[COCA-258]  Ajuste componete tabs

### PR DESCRIPTION
**Pull request by mickaelnn**
### Description
This PR fixes the layout for pbtabs, all this changes was validated with @brunaboeger,
 the list of changes was described on jira card. 
 
 ### Jira
 [COCA-258 - Ajuste componete tabs](https://adapcon.atlassian.net/browse/COCA-258)
 
 ### Screenshots
image
image
image
toma

### CODE REVIEW

**Review by alanneves — CHANGES_REQUESTED** [review]
Could you please show a gif in the vertical/horizontal? We can't see animations in images yet.

**Comment by mickaelnn** [comment]
> Could you please show a gif in the vertical/horizontal? We can't see animations in images yet.

Forgot the gif, sry

**Review by caroltrindade — APPROVED** [review]
LGTM
PS. I think this animation is very slow, but if it was approved by Bruna it's ok

File: src/components/Miscellaneous/Tabs/Tabs.vue [hunk 1]
```diff
@@ -235,6 +242,25 @@ export default {
   },
 
   methods: {
+    colorTab(tab) {
+      return this.isSelectedTab(tab) ? 'primary' : this.tabs[tab]?.color;
+    },
+
+    textAbbreviated(text) {
+      if (this.abbreviatedText) {
+        const textWidth = text.length * 8;
```
Comment (th3rius) [inline src/components/Miscellaneous/Tabs/Tabs.vue:251]: Só uma nota: os caracteres nem sempre vão ter a mesma largura — talvez faça mais faz sentido usar um limite de caracteres fixo mesmo (ao invés de multiplicar por 8)

**Review by th3rius — APPROVED** [review]
os cara trabalhando em primeiro turno

File: src/components/Miscellaneous/Tabs/Tabs.vue [hunk 2]
```diff
@@ -235,6 +242,25 @@ export default {
   },
 
   methods: {
+    colorTab(tab) {
+      return this.isSelectedTab(tab) ? 'primary' : this.tabs[tab]?.color;
+    },
+
+    textAbbreviated(text) {
```
Comment (alanneves) [inline src/components/Miscellaneous/Tabs/Tabs.vue:249]: to work with `...` you should use it:

`.text-overflow-elipsis {
  white-space: nowrap;
  text-overflow: ellipsis;
  overflow: hidden;
}`

**Review by alanneves — CHANGES_REQUESTED** [review]

**Review by alanneves — APPROVED** [review]
LGTM

**Review by planivic1 — COMMENTED** [review]
LGTM

**Pull request merged**